agent/lib/src/commands/ci.dart

```diff
@@ -228,6 +233,15 @@ class ContinuousIntegrationCommand extends Command {
     }
   }
 
+  Future<Null> _unlockKeyChain() async {
+    if (Platform.isMacOS) {
+      await exec(
+        'security',
+        <String>['unlock-keychain', '-p', '\$SECRET', 'login.keychain'],
```

Comment:
Consider rename "SECRET" to something like "FLUTTER_PASS" because there will be more secrets in the future, e.g. fastlane code signing will add a secret to decode encrypted provisioning profiles.

Reply:
Done! updated the env variable name.